services/assets/info.go

```diff
@@ -48,7 +48,7 @@ func (cl *HttpAssetClient) GetCoinInfo(coinId int, token string) (*watchmarket.C
 
 	err = json.Unmarshal(resp.Body(), &info)
```

Comment:
can you rebase fix/tests? On [master there is a `TODO`](https://github.com/trustwallet/watchmarket/blob/master/services/assets/info.go#L49) that we don't need anymore now

Reply:
Yep